lib/code_corps_web/controllers/conversation_controller.ex

```diff
@@ -29,6 +29,18 @@ defmodule CodeCorpsWeb.ConversationController do
     end
   end
 
+  @spec update(Conn.t, map) :: Conn.t
+  def update(%Conn{} = conn, %{"id" => id} = params) do
+    with %Conversation{} = conversation <- Messages.get_conversation(id) |> preload(),
+         %User{} = current_user <- conn |> CodeCorps.Guardian.Plug.current_resource,
+         {:ok, :authorized} <- current_user |> Policy.authorize(:update, conversation),
+         {:ok, %Conversation{} = updated_conversation} <- conversation |> Conversation.update_changeset(params) |> Repo.update
```

Comment:
We should make use of the context here, so instead use `Messages.update_conversation(conversation, params)`.